src/ring_theory/ideal_operations.lean

```diff
@@ -459,6 +468,113 @@ end surjective
 
 end map_and_comap
 
+section jacobson
+variables {R : Type u} [comm_ring R]
+
+/-- The jacobson radical of I is the pullback of the
+jacobson radical of R/I under the quotient map. -/
```

Comment:
The docstring is math-equal to the definition, but not Lean-equal. Does that matter?

Reply:
How about https://github.com/leanprover-community/mathlib/blob/master/src/linear_algebra/basic.lean#L1155

Reply:
There the Lean statement is exactly what the first isomorphism law is, an isomorphism. You are talking about pullbacks and quotient maps, but they don't show up in your Lean-statement (even though these notions exist in Lean).
Why don't you write
```
/-- The jacobson radical of I is the infimum of all maximal ideals containing I. -/
```

Reply:
done